idxbroker/wordpress-plugin
# IMPB-1465 Client's Omnibar search is not looking for sold/closed listings

# Pull Requests

Adjust Omnibar search query to match what is generated by the Listing ID IDX Broker search page so that sold listings are returned when searching via Listing ID.

## Template

### Description of the Change

Add '&a_statusCategory[]=active&a_statusCategory[]=sold' to the end of the search query when the Omnibar searches via Listing ID.

### Verification Process

1. Locate a sold listing within an active MLS on an account
2. Try searching for the sold listing via Listing ID through an Omnibar search on a WordPress site connected via API through IMPress 3.0.10 to the account you're testing with.
3. Note that nothing is returned
4. Apply the fixes in this PR to the plugin
5. Perform the same search again. Note that &a_statusCategory[]=active&a_statusCategory[]=sold' is added to the end of the query to include sold listings when searching via Listing ID.

Zip file of 3.0.10 with the fix applied:
[idx-broker-platinum-3.0.10-with-omnibar-listing-id-fix.zip](https://github.com/idxbroker/wordpress-plugin/files/8773005/idx-broker-platinum-3.0.10-with-omnibar-listing-id-fix.zip)

### Release Notes

Fix: include sold listings when searching via Listing ID through the Omnibar

## Review

Pull Requests must have the sign-off of two other developers and at least one of these must be an IDX Broker team member.
